src/developer/web-api/visualizations.md

```diff
@@ -341,6 +341,8 @@ Table: Visualization attributes
 | subscribers | List of user ids who have subscribed to this Visualization. |
 | translations | Set of available object translation, normally filtered by locale. |
 | outlierAnalysis | Object responsible to keep settings related to outlier analysis. The internal attribute 'outlierMethod' supports: IQR, STANDARD_Z_SCORE, MODIFIED_Z_SCORE. The 'normalizationMethod' accepts only Y_RESIDUALS_LINEAR for now. |
+| seriesKey | Settings related to the series items. |
+| legend | Definitions for the legend keys. |
```

Comment:
These descriptions don't tell me a great deal. They might be fine, but I'd like the opinion of a FE dev, like @janhenrikoverland ?

Reply:
@Philip-Larsen-Donnelly I pushed an update that hopefully makes it a tiny bit more clear.